### DISCUSSION TITLE
Attendees of this school must verify and use their academic email address.

### DISCUSSION THREAD

**Discussion by iadi37** [community/community#52879] [GitHub Education]
I have already given the screenshot of my dated student id to reverify and it clearly mentions my candidature as a student in the respective institute. As my institute doesn't provide with the academic email address I would request to reverify from my student id document.

**Comment by samus-aran** [comment]
Hi @iadi37 👋 

Ensure you are selecting your correct school if you believe your school does not provide academic emails. We enforce students to use academic emails if we have evidence that the institutions distribute them. If you can confirm you are selecting the correct school then we would recommend you reach out to your institution to obtain an academic email.

**Reply by petitsuix** [reply]
Hi @samus-aran ! Sorry to reply here, but I'm not sure opening another ticket is better... 

I'm doing an iOS dev course at Openclassrooms. 
If I select "Openclassrooms" from the very first step, it asks for an "openclassrooms.com" e-mail which I don't have because like iadi37, they don't create such addresses (or at least, at the time I enrolled). So I write "openclassrooms" without the capital "O" to bypass that.

But then, once at the school selection step, on the map, the only school I can choose from is "Openclassrooms Jobs, USA". But mine is in France. Selecting a school is **required** so I select this anyway.

And my application keeps getting rejected because "Attendees of this school must verify and use their academic email address."
It's a nightmare, what should I do ? I had applied once a year ago, and my application had been accepted : the information that I gave the first time was exactly the same and it worked. What's going on ? :(

**Comment by Rishikr15** [comment]
Hi @iadi37 
How did your application then got approved without giving a school issued email id?
I am facing the same issue and the rejection reason is the same.

"Attendees of this school must verify and use their academic email address."

**Comment by AyushKatre05** [comment]
Attendees of this school must verify and use their academic email address. When you have verified your school's academic email address on your account, ensure that you select the school when prompted on the application page. This issue I m facing
